build_version.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import os
+from datetime import datetime, timezone
+
+# To be updated every time we start a new major.minor version.
+major_minor = "1.0"
+
+# Default to 'dev' builds with a YYMMDD build number
+BUILD_TYPE = os.environ.get("BUILD_TYPE") or "dev"
+BUILD_NUMBER = os.environ.get("BUILD_NUMBER") or "YYMMDD"
+
+build_ver = datetime.now(timezone.utc).strftime("%y%m%d")
```

Comment:
This does mean we can't publish more than once in a day, which we should hopefully never need, but if we do, what would be the approach?

Reply:
When you run the pipeline, you can specify a build number for the patch part (in fact, you have to for non dev builds).